go/pkg/api/api.go

```diff
@@ -150,9 +150,9 @@ type TxBody struct {
 	RawTx string `json:"rawTx"`
 }
 
-// swagger:parameters GetAccount GetValidator
+// swagger:parameters GetAccount
 type PubkeyParam struct {
-	// Account address or xpub
+	// Account public key
```

Comment:
throughout the stack we use the pubkey terminology to indicate either an address or xpub as they are both public identifiers. would leave this be and just use the `PubkeyParam` and `PaginatedPubkeyParam` structs for validator endpoint as well